public/app/features/admin/state/apis.tsx

```diff
@@ -21,10 +25,24 @@ export interface ServerStat {
 }
 
 export const getServerStats = async (): Promise<ServerStat | null> => {
-  return getBackendSrv()
-    .get('api/admin/stats')
-    .catch((err) => {
-      console.error(err);
-      return null;
-    });
+  try {
+    // Issue both requests simultaneously
+    const [adminResponse, anonymousStats]: [PromiseSettledResult<ServerStat>, PromiseSettledResult<AnonServerStat>] =
+      await Promise.allSettled([getBackendSrv().get('/api/admin/stats'), getBackendSrv().get('/api/anonymous/stats')]);
```

Comment:
Could we maybe add the anonymous stats directly to the admin stats on backend instead of having 2 separate endpoints?